pkg/metrics/collector.go

```diff
@@ -138,6 +169,30 @@ func (c ResourcesMetricsCollector) collectVulnerabilityReports(ctx context.Conte
 	}
 }
 
+func (c ResourcesMetricsCollector) collectExposedSecretsReports(ctx context.Context, metrics chan<- prometheus.Metric, targetNamespaces []string) {
+	vrList := &v1alpha1.ExposedSecretReportList{}
+	labelValues := make([]string, 7)
+	for _, n := range targetNamespaces {
+		if err := c.List(ctx, vrList, client.InNamespace(n)); err != nil {
+			c.Logger.Error(err, "failed to list exposedsecretreports from API", "namespace", n)
+			continue
+		}
+		for _, vr := range vrList.Items {
```

Comment:
```suggestion
		for _, esr := range vrList.Items {
```

Nit: Another copy-pasted variable name.